release-notes.txt

```diff
@@ -1,3 +1,503 @@
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release:
+===================
+
+The 2021.04 release includes:
+
+- netstats can now collect L1/L2 per neighbor statistics
+- sock DTLS now allows using multiple credentials
+- congure - a framework for implementing congestion control algoritms
+- xfa (cross file arrays) - a mechanism to more easily define stuff across 
+  files (e.g., shell commands)
+- many more modules and drivers have Kconfig support
+- the semtech-loramac package now supports sx126x/llcc68 radios
+- 7 new boards, 5 new sensor drivers, 8 packages upgraded
+
+We've also mostly finished our migration from mailing lists to forum.riot-os.org.
+
+375 pull requests, composed of 992 commits, have been merged since the
+last release, and 16 issues have been solved. 56 people contributed with
+code in 103 days. 2564 files have been touched with 191220 (+) insertions and
+91400 deletions (-).
+
+
+Notations used below:
+=====================
+
+    + means new feature/item
+    * means modified feature/item
+    - means removed feature/item
+
+New features and changes
+========================
+
+Core
+----
++ clist: added clist_is_empty()
++ xfa: intitial support for XFA's (cross file arrays)
+
+System Libraries
+----------------
+* clif: fix attribute parsing
+* shell: allow defining shell commands using XFA
++ dbgpin: module for debugging and profiling
++ ztimer: add ztimer_is_set()
+
+Networking
+----------
+
+* cord: Discovery patch altered to reflect changes in the underlying Resource Directory draft
+* lwip: hook up sam0_eth support
++ congure: framework to implement congestion control and use it in a reusable manner with
+  a number of protocols
++ netstats: L1/L2 per neighbor statistics
++ sock DTLS: allow using multiple credentials
+
+Packages
+--------
+
++ pkg/semtech-loramac: add support for sx126x/llcc68 radios
+* emlearn: bump version to 0.11.6
+* libfixmath: bump version to latest master
+* nanocbor: bump version
+* nanopb: bump version to 0.4.5
+* fatfs: bump version to r0.14a
+* littlefs2: Bump to v2.4
+* monocypher: bump to version 3.1.2
+* openthread: bump version to 20191113
+* wakaama: add patch to cast macro to time_t
+
+Boards
+------
+
+* particle-mesh boards: Use USB serial as stdio by default
+* nucleo-f303re: I2C pins remapped to match the data sheet
+* nucleo-f722ze: Add periph_can support
+* native: compile with `-Og` (was `-O0`)
+
++ New boards:
```

Comment:
Microchip SAMD20 Xplained Pro is missing in this list, it was added by #14662 which is listed in the [2021.04 milestone PRs](https://github.com/RIOT-OS/RIOT/pulls?q=is%3Apr+milestone%3A%22Release+2021.04%22+is%3Aclosed)

Reply:
yup